idflakies-maven-plugin/src/main/java/edu/illinois/cs/dt/tools/plugin/ModuleTestTimeMojo.java

```diff
@@ -15,22 +17,17 @@
 import java.util.Collections;
 import java.util.List;
 
-public class ModuleTestTimePlugin  extends TestPlugin {
+public class ModuleTestTimeMojo  extends AbstractIDFlakiesMojo {
     private String coordinates;
 
-    // Don't delete this.
-    // This is actually used, provided you call this class via Maven (used by the testrunner plugin)
-    public ModuleTestTimePlugin() {
-    }
-
     @Override
-    public void execute(final ProjectWrapper project) {
-        this.coordinates = project.getGroupId() + ":" + project.getArtifactId() + ":" + project.getVersion();
+    public void execute() {
+        this.coordinates = mavenProject.getGroupId() + ":" + mavenProject.getArtifactId() + ":" + mavenProject.getVersion();
```

Comment:
All of these mojos that override execute should call super.execute(), as to get those default things going.